content/en/docs/examples/virtual-machines/bookinfo/index.md

```diff
@@ -12,6 +12,151 @@ aliases:
+    example below:
+
+    {{< text bash >}}
+$ echo "10.55.246.247 productpage.default.svc.cluster.local" | sudo tee -a /etc/hosts
+$ curl -v productpage.default.svc.cluster.local:9080
+< HTTP/1.1 200 OK
+< content-type: text/html; charset=utf-8
+< content-length: 1836
+< server: envoy
+... html content ...
+    {{< /text >}}
+
+The `server: envoy` header indicates that the sidecar intercepted the traffic.
+
+## Running services on the added VM
+
+1. Setup an HTTP server on the VM instance to serve HTTP traffic on port 8080:
+
+    {{< text bash >}}
+    $ gcloud compute ssh ${GCE_NAME}
+    $ python -m SimpleHTTPServer 8080
+    {{< /text >}}
+
+1. Determine the VM instance's IP address. For example, find the IP address
+    of the GCE instance with the following commands:
+
+    {{< text bash >}}
+    $ export GCE_IP=$(gcloud --format="value(networkInterfaces[0].networkIP)" compute instances describe ${GCE_NAME})
+    $ echo ${GCE_IP}
+    {{< /text >}}
+
+1. Add VM services to the mesh
+
+    {{< text bash >}}
+    $ istioctl experimental add-to-mesh external-service vmhttp ${VM_IP} http:8080 -n ${SERVICE_NAMESPACE}
+    {{< /text >}}
+
+    {{< tip >}}
+    Ensure you have added the `istioctl` client to your path, as described in the [download page](/docs/setup/getting-started/#download).
+    {{< /tip >}}
+
+1. Deploy a pod running the `sleep` service in the Kubernetes cluster, and wait until it is ready:
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/sleep/sleep.yaml@
+    $ kubectl get pod
+    NAME                             READY     STATUS    RESTARTS   AGE
+    sleep-88ddbcfdd-rm42k            2/2       Running   0          1s
+    ...
+    {{< /text >}}
+
+1. Send a request from the `sleep` service on the pod to the VM's HTTP service:
+
+    {{< text bash >}}
+    $ kubectl exec -it sleep-88ddbcfdd-rm42k -c sleep -- curl vmhttp.${SERVICE_NAMESPACE}.svc.cluster.local:8080
+    {{< /text >}}
+
+    You should see something similar to the output below.
+
+    {{< text html >}}
+    <!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN"><html>
+    <title>Directory listing for /</title>
+    <body>
+    <h2>Directory listing for /</h2>
+    <hr>
+    <ul>
+    <li><a href=".bashrc">.bashrc</a></li>
+    <li><a href=".ssh/">.ssh/</a></li>
+    ...
+    </body>
+    {{< /text >}}
+
+**Congratulations!** You successfully configured a service running in a pod within the cluster to
+send traffic to a service running on a VM outside of the cluster and tested that
+the configuration worked.
+
+## Cleanup
+
+Run the following commands to remove the expansion VM from the mesh's abstract
+model.
+
+{{< text bash >}}
+$ istioctl experimental remove-from-mesh -n ${SERVICE_NAMESPACE} vmhttp
+Kubernetes Service "vmhttp.vm" has been deleted for external service "vmhttp"
+Service Entry "mesh-expansion-vmhttp" has been deleted for external service "vmhttp"
+{{< /text >}}
+
+## Troubleshooting
```

Comment:
Should this troubleshooting  section come after the existing material about Bookinfo?